subworkflows/local/arg.nf

```diff
@@ -49,14 +55,16 @@ workflow ARG {
         ch_versions = ch_versions.mix( AMRFINDERPLUS_RUN.out.versions )
 
     // Reporting
-        HAMRONIZATION_AMRFINDERPLUS ( AMRFINDERPLUS_RUN.out.report, 'json', AMRFINDERPLUS_RUN.out.tool_version, AMRFINDERPLUS_RUN.out.db_version )
-        ch_versions = ch_versions.mix( HAMRONIZATION_AMRFINDERPLUS.out.versions )
-        ch_input_to_hamronization_summarize = ch_input_to_hamronization_summarize.mix( HAMRONIZATION_AMRFINDERPLUS.out.json )
+        HAMRONIZATION_AMRFINDERPLUS_TSV ( AMRFINDERPLUS_RUN.out.report, 'tsv', AMRFINDERPLUS_RUN.out.tool_version, AMRFINDERPLUS_RUN.out.db_version )
+        HAMRONIZATION_AMRFINDERPLUS_JSON ( AMRFINDERPLUS_RUN.out.report, 'json', AMRFINDERPLUS_RUN.out.tool_version, AMRFINDERPLUS_RUN.out.db_version )
+        ch_versions = ch_versions.mix( HAMRONIZATION_AMRFINDERPLUS_JSON.out.versions )
+        ch_input_to_hamronization_summarize = ch_input_to_hamronization_summarize.mix( HAMRONIZATION_AMRFINDERPLUS_JSON.out.json )
+        ARGNORM_AMRFINDERPLUS ( HAMRONIZATION_AMRFINDERPLUS_TSV.out.tsv.filter{meta, file -> !file.isEmpty()}, 'amrfinderplus', 'ncbi' )
```

Comment:
```suggestion
        HAMRONIZATION_AMRFINDERPLUS ( AMRFINDERPLUS_RUN.out.report, 'tsv', AMRFINDERPLUS_RUN.out.tool_version, AMRFINDERPLUS_RUN.out.db_version )
        ch_versions = ch_versions.mix( HAMRONIZATION_AMRFINDERPLUS.out.versions )
        ch_input_to_hamronization_summarize = ch_input_to_hamronization_summarize.mix( HAMRONIZATION_AMRFINDERPLUS.out.tsv )
        ARGNORM_AMRFINDERPLUS ( HAMRONIZATION_AMRFINDERPLUS.out.tsv.filter{meta, file -> !file.isEmpty()}, 'amrfinderplus', 'ncbi' )
```

Getting rid of the json format, replacing with tsv (also for other tools below).

Reply:
I woudl agree, I don't think it's necessary in this case to double computational tasks/spit out both formats if argNorm can take the TSV.